src/cdk/overlay/public-api.ts

```diff
@@ -24,5 +24,5 @@ export {OverlayPositionBuilder} from './position/overlay-position-builder';
 export {PositionStrategy} from './position/position-strategy';
 export {GlobalPositionStrategy} from './position/global-position-strategy';
 export {ConnectedPositionStrategy} from './position/connected-position-strategy';
-export {FlexibleConnectedPositionStrategy} from './position/flexible-connected-position-strategy';
+export {ConnectedPosition, FlexibleConnectedPositionStrategy} from './position/flexible-connected-position-strategy';
```

Comment:
@jshoudy11 looks like the linter is failing because this line is too long. Can you break it up into multiple lines?